lib/cocoapods-packager/builder.rb

```diff
@@ -49,6 +60,17 @@ def link_embedded_resources
 
     :private
 
+    def build_dynamic_framework(platform, defines, output)
+      UI.puts 'Building dynamic Framework'
+
+      clean_directory_for_dynamic_build
+      if platform.name == :ios
+        build_dynamic_framework_for_ios(platform, defines, output)
+      else
+        build_dynamic_framework_for_mac(platform, defines, output)
```

Comment:
It doesn't build either watchOS or tvOS, yet — but good point, I have created a separate issue for that (#103). It would be good to have OS X support in this PR, so that we retain the functionality we had for static libs.